onadata/apps/api/tests/fixtures/flow-results-responses-list-format.json

```diff
@@ -0,0 +1,9 @@
+{
+    "data": {
+        "type": "flow-results-data",
+        "id": null,
+        "attributes": {
+            "responses": []
+        }
+    }
+}
```

Comment:
I don't think we should have this as an external file - can we make it inline below in the test?
```python
sample_response = {
    "data": {
        "type": "flow-results-data",
        "id": null,
        "attributes": {
            "responses": []
        }
    }
}
...
..
.
self.assertEqual(response.data, sample_response['data'])
```

Reply:
Yes we can. Made the change in the latest commits